lib/src/play_store_search_api.dart

```diff
@@ -2,22 +2,16 @@
  * Copyright (c) 2021 William Kwabla. All rights reserved.
  */
 
+import 'package:html/dom.dart';
 import 'package:html/parser.dart' show parse;
 import 'package:http/http.dart' as http;
 
-class PlayStroeSearchApi {
+class PlayStroeSearchAPI {
   /// Play Store Search Api URL
   final String playStorePrefixURL = 'play.google.com';
 
-  /// Provide an HTTP Client that can be replaced for mock testing.
-  http.Client? client = http.Client();
```

Comment:
By removing `client` you are no longer able to unit test with a mock. I think this should be reverted. See class `ITunesSearchAPI` for an example.

Reply:
I will add it back